### PR TITLE
added package.json

### PR DESCRIPTION
**Pull request by nagyadam2092**
Hello!

package.json is needed if someone wants to reach this repository via npm, so I added it.
Please check the information I wrote there.

Thank you, 
Adam

### CODE REVIEW

**Comment by nagyadam2092** [comment]
any chance to merge this request soon?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/piwik/piwik/issues"
+    },
+    "keywords": [
+        "piwik",
+        "liberating",
+        "web",
+        "analytics"
+    ],
+    "dependencies": {},
+    "license": "GPL v3"
```
Comment (tsteur) [inline package.json]: Can we call license: `GPLv3 or later`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Piwik",
+    "version": "2.16.0",
+    "description": "Liberating Web Analytics. Piwik is the leading open alternative to Google Analytics that gives you full control over your data. Piwik lets you easily collect data from websites, apps & the IoT and visualise this data and extract insights. Privacy is built-in. ",
+    "author": "http://piwik.org/",
```
Comment (tsteur) [inline package.json]: Maybe we use `https` here?

**Comment by tsteur** [comment]
In our `bower.json` we have some files to ignore specified. Like 

```
"ignore": [
    "**/.*",
    "node_modules",
    "tests"
  ]
```

Is this possible with NPM as well? Or maybe we live the tests there as it will be only used for development via NPM. We always recommend to download the prepared ZIP file from piwik.org for deploying Piwik on production systems since it makes sure to remove test files and unneeded plugins etc.

Also, we might need to give some instructions on how to proceed with Piwik installation maybe? Eg Piwik won't work after fetching Piwik via NPM as it will be still needed to run `Composer` to fetch all PHP dependencies specified here. https://github.com/piwik/piwik/blob/2.16.0/composer.json#L38-L60

**Comment by mattab** [comment]
(fyi @nagyadam2092  this FAQ lists instructions how to initialise Piwik when fetched from Git http://piwik.org/faq/how-to-install/faq_18271/ )

**Comment by mattab** [comment]
Hi @nagyadam2092 
could you maybe check the review/feedback above so we can consider merging your PR?

**Comment by mattab** [comment]
Thank you for this proposed pull request.

Because it was last updated more than one month ago, it is our policy to close pull requests opened for a long time without updates. If you would like to continue work on the pull request, please simply ping us to have it re-opened (after you have pushed a new commit).

We hope you understand this and we look forward to seeing an update from you on this pull request or another one!

Thanks.

**Pull request closed without merging**